astropy/stats/tests/test_funcs.py

```diff
@@ -256,6 +256,8 @@ def test_biweight_midcovariance():
     cov = funcs.biweight_midcovariance(d)
     std = np.around(np.sqrt(cov.diagonal()), 1)
     assert_allclose(std, [ 2.6,  2.1,  1.6,  1.5,  1.9])
+    # Test 3: Ensure matrix is symmetric
+    assert_allclose(cov, cov.T)
```

Comment:
@nkern This isn't a sufficient regression test because it passes even with the old (buggy) code.  You need to update the input data such that it fails with the old (buggy) code (i.e. gives an asymmetric result), but passes with the new (fixed) code.